src/components/Sum.jsx

```diff
@@ -1 +1,9 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum(props) {
+  const { a = 0, b = 0 } = props;
```

Comment:
```suggestion
export function Sum({ a = 0, b = 0 }) {
```
You can immediately get them using [destructuring](https://www.geeksforgeeks.org/destructuring-of-props-in-reactjs/)